src/main/java/art/heredium/domain/post/entity/Post.java

```diff
@@ -45,6 +45,9 @@ public class Post extends BaseEntity {
   @Column(name = "name", nullable = false)
   private String name;
 
+  @Column(name = "original_file_name", nullable = false)
+  private String originalFileName;
```

Comment:
```suggestion
  @Column(name = "image_original_file_name", nullable = false)
  private String imageOriginalFileName;
```
Thêm cái này để clear hơn.

Reply:
I updated in https://github.com/Incuvers-IT/heredium-BE/pull/41/commits/607164b8ffcdc19c49a50ae1b6a9ea6c3e2e81a3